src/store/store.ts

```diff
@@ -1,6 +1,6 @@
 import { configureStore } from "@reduxjs/toolkit";
 import { setupListeners } from "@reduxjs/toolkit/query";
-import { baseApi } from "containers/home/api";
+import  baseApi from "services/api";
```

Comment:
I think we have alias, but lets make it like @services

Reply:
Added alias.